package.json

```diff
@@ -14,8 +14,9 @@
     "@babel/preset-env": "^7.12.7",
     "@rails/ujs": "^6.0.3-4",
     "@rails/webpacker": "^5.2.1",
-    "algoliasearch": "^4.8.2",
     "@vonagevolta/volta2": "^0.1.5",
+    "alex": "^9.1.0",
```

Comment:
Alex is a dev dependency, not a runtime dependency

Reply:
Good catch! I'll fix.

Reply:
Done :)